### DISCUSSION TITLE
How to improve NER + RELATION model Predictions, when trained together?

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#9229] [Help: Coding & Implementations] [answered]
Hi,
I have trained a NER + RELATION model together. And I achieved an overall accuracy of 91-92% for Both NER and RELATION. But when I predict the output for RELATION between the entities for the new dataset, I can see there are very less number of predictions with higher confidence of 0.8 and more number of predictions with lower confidence which ranges from 0.0001-0.49. The below image shows the highlighted example for which the relation is correct but confidence is very low. Please advise me to increase the prediction confidence as it is very low even after achieving overall 90+ accuracy. What changes do I need to do? Please advise something so that I can improve these results. 
I have used 154 training documents and 35 test documents for this training.
![image](https://user-images.githubusercontent.com/49562460/133619028-9ac082b9-0a2f-4d2f-a871-df544a6be689.png)

**Comment by svlandeg** [comment, marked as answer]
This question is similar to https://github.com/explosion/spaCy/discussions/9220 and I'll answer there.
